lingpy/convert/strings.py

```diff
@@ -403,24 +405,16 @@ def multistate2nex(taxa, matrix, filename='', missing="?"):
 
 
 def write_nexus(
-        taxa, matrix, custom=None, custom_name='lingpy', missing="?", gap="-",
-        template="mrbayes.nex", filename="mrbayes.nex", dtype="RESTRICTION",
-        symbols="10",
-        commands=None, commands_name="mrbayes"):
+        wordlist, ref="cogid", custom=None,
+        custom_name='lingpy', missing="?", gap="-", template="mrbayes.nex",
```

Comment:
Do we want to hide the template implementation from the user e.g. instead of `template`, have 
`nexus_type = ('mrbayes', 'beast', 'beastwords')`. 

The correct template could be chosen based on that argument?

Reply:
Yes, this seems to be an elegant solution! It would, of course, deprive people who want to flexibly work with this code from the possibility, but on the other hand: those who want flexibility can write their nexus export themselves, the code is not that long to adjust it (and it's almost a standalone).